HACKER/HACKER/Resources/Extensions/UIFont+Extension.swift

```diff
@@ -38,7 +50,11 @@ extension UIFont {
     return UIFont(name: "NotoSansKR-Regular", size: 14.0)!
   }
   
-  class var btnText: UIFont {
+  class var btnText32: UIFont {
     return UIFont(name: "KCC-Kimhwanki", size: 32.0)!
   }
+  
+  class var btnText40: UIFont {
+    return UIFont(name: "KCC-Kimhwanki", size: 40.0)!
+  }
```

Comment:
이게 나중에 폰트 사이즈가 많아지면 관리하기가 너무 힘들어져서 
이런 식으로 size를 인자로 받는 형태로 익스텐션을 만들기도 해
사용도 그냥 `btnText32` 대신 `.btnText(size: 32)` 이렇게 할 수 있고

3/23 6:52 수정 
+) 대학 수업 듣다가 갑자기 생각났는데 애플이 기본제공하는 system font가 `.systemFont(ofSize: 10)` 이런 식으로 돼있어서 그 형식 맞춰주면 더 간지나겠다 `.btnText(ofSize: 32)` 이렇게
```suggestion
  class func btnText(ofSize: CGFloat) -> UIFont {
      return UIFont(name: "KCC-Kimhwanki", size: size)!
  }
```